content/guards.md

```diff
@@ -202,8 +202,17 @@ export class RolesGuard implements CanActivate {
     }
     const request = context.switchToHttp().getRequest();
     const user = request.user;
-    return matchRoles(roles, user.roles);
+    return this.matchRoles(roles, user.roles);
```

Comment:
Thanks for adding this. I don't see value in adding a private function here. Developers have there own logic to `matchRoles`.

I would rather just re-write this to:

```
return matchRoles(roles, user.roles); // Custom function to match roles
```